queries/scala/highlights.scm

```diff
@@ -77,14 +106,17 @@
 (boolean_literal) @boolean
 (integer_literal) @number
 (floating_point_literal) @float
-(string) @string
 
 [
 (symbol_literal)
 (string)
 (character_literal)
+(interpolated_string_expression)
 ] @string
 
+(interpolation "$" @punctuation.special)
+(interpolation (identifier) @variable)
```

Comment:
Can't this be treated like interpolations in Python or Bash? The variables should get `@variable` automatically when the string highlight is removed and the interpolation query is placed on the top of the file. Maybe this does not work for Scala

Reply:
Yes, I was wondering what's the best approach for interpolation. I'll try what you proposed.

Reply:
It's looking great with `@none` now, thanks!